cpp/src/indexing/construction/detail/phase_2.cuh

```diff
@@ -193,6 +193,8 @@ inline rmm::device_uvector<uint32_t> compute_parent_positions(
                            position_map.begin());
     // line 2 of algorithm in Fig. 5 in ref.
     rmm::device_uvector<uint32_t> parent_pos(num_child_nodes, stream);
+    thrust::uninitialized_fill(rmm::exec_policy(stream)->on(stream),
```

Comment:
Run clang-format on your code before submitting.  

```
python cpp/scripts/run-clang-format.py
```

Reply:
I fixed the lint and merged the build fixes in.

Reply:
Thanks Paul. Having been busy in the past few days before coming back.